var/spack/repos/builtin/packages/py-batchspawner/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyBatchspawner(PythonPackage):
+    """This is a custom spawner for Jupyterhub that is designed for
+    installations on clusters using batch scheduling software."""
+
+    homepage = "https://github.com/jupyterhub/batchspawner"
+    pypi     = "batchspawner/batchspawner-1.0.1.tar.gz"
```

Comment:
```suggestion
    pypi     = "batchspawner/batchspawner-1.1.0.tar.gz"
```

Reply:
Done, but I thought these kind of offsets don't matter. Has this changed?